backend/src/model/van.py

```diff
@@ -0,0 +1,11 @@
+from sqlalchemy import ForeignKeyConstraint
+from sqlalchemy.orm import Mapped, mapped_column
+from src.db import Base
+
+
+class VanModel(Base):
+    __tablename__ = "vans"
+    id: Mapped[int] = mapped_column(
+        primary_key=True, autoincrement=True, nullable=False
+    )
+    wheelchair: Mapped[bool] = mapped_column(nullable=False)
```

Comment:
In alembic I see this table as having route_id int it.

Reply:
Used old copies of alembic tables, it seems